.circleci/config.yml

```diff
@@ -99,6 +99,47 @@ commands:
             python ./test_runner.py --suite=push -o $ENZOTEST_DIR --answer-name=$ANSWER_NAME --local --strict=high --verbose << parameters.flags >>
           fi
 
+  compile-test:
+    description: "Test that enzo compiles in a few different configurations."
+    steps:
+      - run: |
+          source $BASH_ENV
+          source $HOME/venv/bin/activate
+          ./configure
+          cd src/enzo
+          make machine-linux-gnu
+          make load-config-test-suite
+          # test various baryon/particle precisions
+          for prec in 32 64
+          do for part in 32 64 128
+          do
+          if [ $prec == $part ]; then
+          continue
+          fi
+          make precision-$prec
+          make particles-$part
+          make clean
+          make -j 4
+          done ; done
+          # return this to default
+          make particles-64
```

Comment:
I would add "make precision-64" to return that to default as well (I know the above loop should leave it there, but what if that loop got changed) -- also this just makes it clearer.

Reply:
Sure thing.